testimonials/views.py

```diff
@@ -29,7 +29,10 @@ class AttestationViewSet(ReadOnlyModelViewSet):
     """Viewset for attestations."""
 
     serializer_class = AttestationSerializer
-    queryset = Attestation.objects.all()
+    queryset = Attestation.objects.filter(
+        Q(publish_date__isnull=True) | Q(publish_date__lte=now_in_utc())
+    ).all()
     pagination_class = LargePagination
     filter_backends = [MultipleOptionsFilterBackend]
     filterset_class = AttestationFilter
+    permission_classes = [AllowAny]
```

Comment:
The permission class `main.permissions.AnonymousAccessReadonlyPermission` has been used for other read-only views to allow anonymous access (like `TopicViewSet`, `OfferedByViewset`), but this works too.